<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChessGame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background:rgb(50,46,43);
}
QPushButton{
	color:rgb(162,150,135);
	background-color:rgb(255,255,255);
	font-size:48px;
	border-radius:10px;
}
QPushButton::hover{
	background-color:rgb(222,222,222);
}
QFrame{
	background-color:rgb(222,222,222)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="moves">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>90</y>
      <width>311</width>
      <height>491</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>540</width>
      <height>540</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color rgb(255,255,255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>25</y>
       <width>490</width>
       <height>490</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(50,46,43);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="backToTasksButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>640</y>
      <width>230</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="chess_img">
    <property name="geometry">
     <rect>
      <x>-156</x>
      <y>18</y>
      <width>670</width>
      <height>740</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>chess.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>100</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>статус</string>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>150</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>chess_img</zorder>
   <zorder>moves</zorder>
   <zorder>frame</zorder>
   <zorder>backToTasksButton</zorder>
   <zorder>label</zorder>
   <zorder>status</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
